src/test/resources/pages/InstructorSearchPageSearchCommentsAsHelper.html

```diff
@@ -73,7 +73,11 @@
                            <b>
                               Anonymous
                            </b>
-                           [Sat, 02 Jun 2012, 07:21 PM UTC]
+                           [
+                           <span class="client-time">
```

Comment:
@damithc it is to ease the detection on which timestamp needs to be updated to the client's timezone. I will elaborate further on your comment as why we don't do it on the server side.